keycastr/KCKeyboardTap.m

```diff
@@ -87,7 +87,20 @@ -(BOOL) installTapWithError:(NSError **)error {
     if (tapInstalled) {
         return YES;
     }
-    
+
+    // 10.9 and after
+    if (AXIsProcessTrustedWithOptions != NULL) {
+        NSDictionary *options = @{(__bridge id) kAXTrustedCheckOptionPrompt : @YES};
+        BOOL accessibilityEnabled = AXIsProcessTrustedWithOptions((__bridge CFDictionaryRef) options);
+
+        if (!accessibilityEnabled) {
+            if (error != NULL) {
+                *error = [self constructErrorWithDescription:@"Could not create keyDown event tap!"];
+            }
+            return NO;
```

Comment:
This results in the app exiting, which tears down the native dialog generated by `AXIsProcessTrustedWithOptions()`. In order to use this API (which we should do!) we need to refactor to wait to exit the app until the user interacts with that dialog, if possible.